test/globalSetup.ts

```diff
@@ -33,7 +33,7 @@ module.exports = async () => {
   await exec("docker start localsecret");
 
   await waitForBlocks();
-
+  await sleep(5000);
```

Comment:
Why?

Reply:
Temporary thing to see if it fixes the CIs that depend on it